src/components/walkthroughResources/walkthroughResources.js

```diff
@@ -63,13 +68,43 @@ class WalkthroughResources extends React.Component {
     return provisioningStatus;
   }
 
+  buildResourcesList() {
+    const resources = this.mapServiceLinks();
+    let resourceList = null;
+    if (resources && resources.length > 0) {
+      resourceList = resources.map(resource => {
+        console.log(resource);
+        return (
+          <div key={resource.title}>
+            <h4 className="integr8ly-helpful-links-product-title">
+              {resource.statusIcon}
+              &nbsp;
+              {resource.title}
+              &nbsp;
+              {resource.gaStatus === 'community' ? (
+                <span className="integr8ly-label-community integr8ly-walkthrough-labels-tag">community</span>
+              ) : (
+                <span />
+              )}
+              {resource.gaStatus === 'preview' ? (
+                <span className="integr8ly-label-preview integr8ly-walkthrough-labels-tag">preview</span>
+              ) : (
+                <span />
+              )}
+            </h4>
+            <div className="list-unstyled" dangerouslySetInnerHTML={{ __html: resource.html }} />
```

Comment:
The html generated from the adoc is actually an `ul` and that's what ends up inside this div. Unfortunately we don't have control over the generated html, so i can't apply the class to the `ul` directly. That's why i'm using this wrapper div.

Reply:
Alrighty, makes sense. Then the class should be added correctly?

Reply:
I think we don't even need this class because the rule is applied to `integreatly-module-frame`: https://github.com/integr8ly/tutorial-web-app/blob/17bf549b7622be839631b3ca8f6b232733132bec/src/styles/application/_taskLayout.scss#L123

I'll update this.

Reply:
We shouldn't be targeting the HTML element directly - ideally, we would use the class `list-unstyled` on the `<ul>` and then delete the CSS around the nested `<ul>`.

Looking at the PR locally, this is what I'm seeing for generated HTML, which places the `list-unstyled` class on the wrong element.
![screen shot 2018-11-27 at 1 18 35 pm](https://user-images.githubusercontent.com/4032718/49102522-06791b00-f247-11e8-9c0b-8990e03ca8d3.png)

Reply:
Instead of adding another dependency, what about replacing the `list-unstyled` on the `<div>` with a custom class (maybe `integr8ly-resources-list` ?) and use that instead of targeting the `<ul>` directly in the `.scss` file? That way, the styles would only apply the element in the class and then we don't lock the sidebar into all `<ul>`s not having bullets.

Reply:
Ah, i think i got you now. I've created a specific class for this: `integr8ly-helpful-resources-list` instead of targetting all uls in the side panel.

Reply:
Looks good, thanks for making the updates!